src/wallet/wallet.h

```diff
@@ -805,7 +805,7 @@ class CWallet final : public WalletStorage, private interfaces::Chain::Notificat
     bool IsUsedDestination(const uint256& hash, unsigned int n) const EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
     void SetUsedDestinationState(WalletBatch& batch, const uint256& hash, unsigned int n, bool used, std::set<CTxDestination>& tx_destinations) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
 
-    std::vector<OutputGroup> GroupOutputs(const std::vector<COutput>& outputs, bool single_coin) const;
+    std::vector<OutputGroup> GroupOutputs(const std::vector<COutput>& outputs, bool single_coin, const size_t max_ancestors = 10000) const;
```

Comment:
Where does the value of `10000` come from? Could it be documented and maybe hoisted to a static constant?

Reply:
I think this should be based on the wallet-determined max ancestor count, and should not have a default value here. (i.e. I suggest removing ` = 10000`)

Reply:
Right, `(gArgs.GetArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT)` which defaults to 25.

Reply:
Yep!

Reply:
I removed the default value (10000), thanks!